app/src/main/java/com/chess/clock/fragments/TimeControlFragment.java

```diff
@@ -117,7 +144,8 @@ public void onAttach(Activity activity) {
             mListener = (OnTimeControlListener) activity;
 
             // Fetch current TimeControl object
-            mTimeControl = mListener.getEditableTimeControl();
+            mTimeControlWrapper = mListener.getEditableTimeControl();
+            Log.e("", mTimeControlWrapper.toString());
```

Comment:
Let's say something other than an empty string here please...actually...does this need to be logged, and does this need to be logged as an error?  I'm slightly confused about this line now.

Reply:
This was for my debugging purposes. I'll remove this.